README.md

```diff
@@ -28,6 +28,13 @@ As the application is being developed, every Pull Request has its own test
 environment automatically built and deployed to. The path to the environment
 is added as a comment to the PR in GitHub.
```

Comment:
This is no longer true - the link is in the checks (as the Details for rancher/deployment check).

The readme has not been updated to reflect this change.

Reply:
Add as an issue.

Reply:
My bad. Sorted this and alphabetised the docker config to make it easier to search through.